tests/general-api-tests/axis.spec.ts

```
@@ -0,0 +1,384 @@
+            .soft(
+              value,
+              `Step value (${value}) should not be null for ${actualQuotient}`
+            )
+            .not.toBeNull();
+          expect
+            .soft(
+              step.label,
+              `Step label (${step.label}) should be defined for ${actualQuotient}`
+            )
+            .toBeDefined();
+          if (value === 0) acc++;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)